web/private/scripts/drush_config_distro_update.php

```diff
@@ -2,8 +2,11 @@
 
 // Ensure update mode is 'merge' and run config-distro-update.
 echo "Running Config Distro update...\n";
-passthru('drush -n -y state:set config_sync.update_mode 1 --input-format=integer');
-passthru('drush -n -y config-distro-update');
+passthru('drush -n -y config-distro-update --update-mode=1');
+echo "Config Distro update complete.\n";
+
+echo "Syncing permissions...\n";
+passthru('drush -n -y az-core-config-add-permissions');
 echo "Config Distro update complete.\n";
```

Comment:
Might be better to make the permissions sync a separate script.   Either way, this line should probably change.
```suggestion
echo "Permissions sync complete.\n";
```